### PR TITLE
close #7 add properties to space

### LINKED ISSUE #7: Add properties to space
As a host,
in order to promote my space more clearly,
I want to add details to space.

### PR DESCRIPTION
**Pull request by robota-x**
any listed space now has price, name, description. no validation on the model yet

### CODE REVIEW

File: app/controller/spaces.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Bnb < Sinatra::Base
 
   get '/spaces' do
-    @spaces = Space.all
+    p @spaces = Space.all
```
Comment (kylebuttner) [inline app/controller/spaces.rb]: remove `p`

**Comment by ccfz** [comment]
:shipit:

**Comment by robota-x** [comment]
closed #7 add properties to space

File: app/controller/spaces.rb [hunk 2]
```diff
@@ -12,11 +11,12 @@ class Bnb < Sinatra::Base
 
   post '/spaces/new' do
 
-    space = Space.new(name: params[:space_name])
-   
+    space = Space.create(name: params[:space_name],
```
Comment (kylebuttner) [inline app/controller/spaces.rb]: `Space.create` should be `Space.new`

File: app/controller/spaces.rb [hunk 3]
```diff
@@ -12,11 +11,12 @@ class Bnb < Sinatra::Base
...
+      description: params[:space_description],
+      price: params[:space_price].to_f.round(2)
+      )
     space.user = User.get(session[:user_id])
-    space.save   
-
+    space.save
```
Comment (kylebuttner) [inline app/controller/spaces.rb:19]: Should `space = space.save`? Won't `space` always be true if it's being assigned above as `Space.new`?

Comment (robota-x) [inline app/controller/spaces.rb:19]: moved to bugs

**Pull request merged**